### PR TITLE
creates a module that translates between english and pig latin

### PR DESCRIPTION
**Pull request by badmckinney**

### CODE REVIEW

File: pig-latin.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+          if (vowels.includes(word[i].toLowerCase())) {
+            let pre = word.slice(i);
+            let post = word.slice(0, i);
+            word = `${pre}-${post}ay`;
+            translated.push(word);
+            return;
+          }
+        }
+      }
+      return words;
```
Comment (taesup) [inline pig-latin.js]: this line will never be executed because both the if and the else have a return.

File: pig-latin.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    let translated = [];
+    words.forEach(word => {
+      if (vowels.includes(word[0].toLowerCase())) {
+        word += "-ay";
+        translated.push(word);
+        return;
+      } else {
+        for (let i = 0; i < word.length; i++) {
+          if (vowels.includes(word[i].toLowerCase())) {
+            let pre = word.slice(i);
```
Comment (taesup) [inline pig-latin.js:20]: The variable names here are actually the opposite of what they claim to be. (pre, post)

File: pig-latin.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            let post = word.slice(0, i);
+            word = `${pre}-${post}ay`;
+            translated.push(word);
+            return;
+          }
+        }
+      }
+      return words;
+    });
+    console.log(translated.join(" "));
```
Comment (taesup) [inline pig-latin.js]: Remove console.log

File: pig-latin.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    return translated.join(" ");
+  };
+
+  //Translate Pig Latin to English
+  const translatePig = (str) => {
+    debugger;
```
Comment (taesup) [inline pig-latin.js]: Remove debugger statement.

File: test/pig-latin-spec.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
+const translator = require('./../pig-latin.js');
+const chai = require('chai');
+const should = chai.should();
+
+console.log(translator().translateEng);
```
Comment (taesup) [inline test/pig-latin-spec.js]: Remove console.log

File: pig-latin.js [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+        return;
+      } else {
+        let post = word.slice(0, hyphen);
+        let pre = word.slice(hyphen + 1, word.length - 2);
+        word = `${pre}${post}`;
+        translated.push(word);
+        return;
+      }
+    });
+    console.log(translated.join(" "));
```
Comment (taesup) [inline pig-latin.js]: Remove console.log

**Review by taesup — CHANGES_REQUESTED** [review]

File: pig-latin.js (hunk 2 above)
Comment (badmckinney) [reply pig-latin.js:20]: I named the variables based on what they would be in the translated word, should I reverse it?

Comment (taesup) [reply pig-latin.js:20]: I think it's best to name it in the context of the function. That way as you're reading through the code it makes more sense, especially if this was a larger function or a function is a larger code base.

**Pull request merged**